docs/reference/aggregations/bucket/histogram-aggregation.asciidoc

```diff
@@ -286,3 +286,93 @@ POST /sales/_search?size=0
 // TEST[setup:sales]
 
 <1> Documents without a value in the `quantity` field will fall into the same bucket as documents that have the value `0`.
+
+[[search-aggregations-bucket-histogram-aggregation-histogram-fields]]
+==== Histogram fields
+
+Running a histogram aggregation over histogram fields computes the total number of counts for each interval.
+
+For example, executing a histogram aggregation against the following index that stores pre-aggregated histograms
+with latency metrics (in milliseconds) for different networks:
+
+[source,console]
+--------------------------------------------------
+PUT metrics_index/_doc/1
+{
+  "network.name" : "net-1",
+  "latency_histo" : {
+      "values" : [1, 3, 8, 12, 15],
+      "counts" : [3, 7, 23, 12, 6]
+   }
+}
+
+PUT metrics_index/_doc/2
+{
+  "network.name" : "net-2",
+  "latency_histo" : {
+      "values" : [1, 6, 8, 12, 14],
+      "counts" : [8, 17, 8, 7, 6]
+   }
+}
+
+POST /metrics_index/_search?size=0
+{
+    "aggs" : {
+        "latency_buckets" : {
+            "histogram" : {
+                "field" : "latency_histo",
+                "interval" : 5
+            }
+        }
+    }
+}
+--------------------------------------------------
+
+
+The `histogram` aggregation will sum the counts of each interval computed based on the `values` and
+return the following output:
+
+[source,console-result]
+--------------------------------------------------
+{
+    ...
+    "aggregations": {
+        "prices" : {
+            "buckets": [
+                {
+                    "key": 0.0,
+                    "doc_count": 18
+                },
+                {
+                    "key": 5.0,
+                    "doc_count": 48
+                },
+                {
+                    "key": 10.0,
+                    "doc_count": 25
+                },
+                {
+                    "key": 15.0,
+                    "doc_count": 6
+                }
+            ]
+        }
+    }
+}
+--------------------------------------------------
+// TESTRESPONSE[skip:test not setup]
+
+[IMPORTANT]
+========
+Histogram aggregation is a bucket aggregation, which partitions documents into buckets rather than calculating metrics over fields like
```

Comment:
I thought about it some more and can't think of a better/clearer way to express this concept... so :+1: and maybe we can tweak it later if we think of a better way.  I think it's clear what the limitations are so I'm not too concerned.